ports/nrf/boards/blyst_nano/mpconfigboard.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * This file is part of the Micro Python project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 Glenn Ruben Bakke
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#define MICROPY_HW_BOARD_NAME       "BLYST Nano"
+#define MICROPY_HW_MCU_NAME         "NRF52832"
+#define MICROPY_PY_SYS_PLATFORM     "IDK-BLYST-NANO"
+
+#define MICROPY_PY_MACHINE_SOFT_PWM (1)
+#define MICROPY_PY_MUSIC            (1)
+
+#define MICROPY_PY_MACHINE_UART     (1)
+#define MICROPY_PY_MACHINE_HW_PWM   (1)
+#define MICROPY_PY_MACHINE_HW_SPI   (1)
+#define MICROPY_PY_MACHINE_TIMER    (1)
+#define MICROPY_PY_MACHINE_RTCOUNTER (1)
+#define MICROPY_PY_MACHINE_I2C      (1)
+#define MICROPY_PY_MACHINE_ADC      (1)
+#define MICROPY_PY_MACHINE_TEMP     (1)
+#define MICROPY_PY_RANDOM_HW_RNG    (1)
+
+#define MICROPY_HW_HAS_LED          (1)
```

Comment:
A brief look online, and i could not see any LEDs on this module. If this is set to 1, todays main.c for the nrf port will set the LED1 ON to indicate it is alive. 
Ref: https://github.com/micropython/micropython/blob/master/ports/nrf/main.c#L195

I would recommend to at least set this to 0, or even remove all the lines in this file that relates to LED, even the help text in the bottom which is elided if MICROPY_HW_HAS_LED is set to 0.
https://github.com/micropython/micropython/blob/master/ports/nrf/help.c#L40

It's a bit hard to know what LED's are available on system where this module is mounted, so maybe you would like to keep the LED defines, but disable them?

Reply:
@glennrub Thanks for reviewing.  The module is generic but this port is for 2 boards: a breakout board which has 3 LED & 2 buttons, a DevKit board which has 4 LED and 2 buttons + a SWD debugger.  The LED1 does effectively turn on when it is running.  

Thanks for the link to download the softdevice.  I will modify the script.  I didn't know where to get those links.  If you have the link for the S140 I can make the change as well for both.

The IBK-BLYST-NANO breakout board
![img_3437_crop](https://user-images.githubusercontent.com/6577947/52603044-085c7e80-2e33-11e9-891f-307813d99469.jpeg)

The IDK-BLYST-NANO DevKit board.
![img_3451_90_crop](https://user-images.githubusercontent.com/6577947/52603053-14484080-2e33-11e9-8450-72b69f50edd1.png)

Reply:
@glennrub Do I get your shell script and put it in micropython ? How should I do this ?

Reply:
@hnhoan, thanks for the elaboration about the LEDs and Buttons. So it looks pretty much like they are pin compatible to me, except this one LED that differs. The alternative to get this LED count correct would be to add two boards, but what you have suggested is from my point of view a good compromise to prevent to much duplication.

I would not mind if you change the copyright in this board definition file to your own name. If you want to of course, else keep it as is.

Reply:
@glennrub I did begin to split the pull.  The first pull is to add support for SD6.1.1 which I already created a new pull.  The second coming pull will be adding the boards.  This pull can discarded in favour of the 2 new pulls.  Thanks.